templates/overrides/navigation/block--system-branding-block.html.twig

```diff
@@ -16,14 +16,10 @@
 {% block content %}
   <div class="d-flex align-items-center my-3">
   {% if site_logo %}
-    <a href="{{ path('<front>') }}" rel="home" class="site-logo">
```

Comment:
the fact that it's not linked is a mistake on the FRONT side, please put back and create a ticket for them